src/main/java/io/supertokens/useridmapping/AppIdentifierStorageAndUserIdMapping.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *    Copyright (c) 2023, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ *    This software is licensed under the Apache License, Version 2.0 (the
```

Comment:
this should go in root level alongside Main.java? cause it's not a thing specific to useridmapping anymore

Reply:
Done